packages/eventual-send/src/local.js

```diff
@@ -0,0 +1,39 @@
+const q = JSON.stringify;
+
+const ntypeof = specimen => (specimen === null ? 'null' : typeof specimen);
+
+export const localApplyFunction = (t, args) => {
+  if (!(t instanceof Function)) {
+    const ftype = ntypeof(t);
+    throw TypeError(
+      `Cannot invoke target as a function; typeof target is ${q(ftype)}`,
+    );
+  }
+  return t(...args);
+};
+
+export const localApplyMethod = (t, method, args) => {
+  if (method === undefined || method === null) {
+    return localApplyFunction(t, args);
+  }
+  if (t === undefined || t === null) {
+    const ftype = ntypeof(t);
+    throw TypeError(
+      `Cannot deliver ${q(method)} to target; typeof target is ${q(ftype)}`,
+    );
+  }
```

Comment:
For just `null` or `undefined`, `typeof` is overkill.

```suggestion
  if (t === undefined || t === null) {
    throw TypeError(
      `Cannot deliver ${q(method)} to target; target is ${q(t)}`,
    );
  }
```

Reply:
Now that I'm using the `assert` global, this would be a change in the error messages, and I still don't like introducing the ambiguity of `target is "[undefined]"` versus `typeof target is "undefined"`.

It's also not in the hot path, so I'm not concerned about overkill.

Reply:
ok. resolving.